code/LeftAndMain.php

```diff
@@ -683,6 +683,10 @@ protected function init()
         // file because insufficient information exists when that is being processed
         $htmlEditorConfig = HTMLEditorConfig::get_active();
         $htmlEditorConfig->setOption('language', TinyMCEConfig::get_tinymce_lang());
+        $langUrl = TinyMCEConfig::get_tinymce_lang_url();
+        if ($langUrl) {
+            $htmlEditorConfig->setOption('language_url', $langUrl);
+        }
```

Comment:
Along with the language itself, we need to point to the correct translation file.